watchdog/readconfig.go

```diff
@@ -28,16 +28,19 @@ func parseBoolValue(val string) bool {
 	return false
 }
 
-func parseIntValue(val string) int {
+func parseIntOrDurationValue(val string, fallback time.Duration) time.Duration {
```

Comment:
fallback better called default for clarity

Reply:
very subjective, but I'd be inclined to call that function `toDuration`... I suck at naming tho

Reply:
I took `fallback` name from other projects. I will make it `default`.

Reply:
This has caught me before.... `default` is a [golang keyword](https://golang.org/ref/spec#Keywords)

Reply:
how about `defaultValue` ?